demo/x-select.html

```diff
@@ -21,15 +21,15 @@
       }
     </style>
     <div on-tap="open">
-      <content select=".dropdown-trigger"></content>
+      <slot name="dropdown-trigger"></slot>
     </div>
     <iron-dropdown id="dropdown"
       vertical-align="[[verticalAlign]]"
       horizontal-align="[[horizontalAlign]]"
       disabled="[[disabled]]"
       open-animation-config="[[openAnimationConfig]]"
       close-animation-config="[[closeAnimationConfig]]">
-      <content select=".dropdown-content"></content>
+      <slot></slot>
```

Comment:
I saw this as an occasion to update `iron-dropdown` to be less selective, but since that could be seen as a breaking change (e.g. [this example](https://github.com/PolymerElements/paper-menu-button/pull/105/files#r110748452)) it's probably best to keep that strict selectiveness via a named slot. Will update